src/components/Callout/__tests__/styles.test.tsx

```diff
@@ -1,9 +1,24 @@
 import Callout from '../styles';
 import { render, screen } from '@testing-library/react';
+import createCache from '@emotion/cache';
+import { CacheProvider } from '@emotion/react';
+
+function renderWithProvider(ui) {
```

Comment:
Wrapping the components with `CacheProvider` allows us to keep using `first-child`.